CMake/python.cmake

```diff
@@ -15,15 +15,18 @@ function(TEST_PYTHON_MODULE MODULE_NAME MODULE_PRESENT)
 endfunction()
 
 # Test python module prerequisites for a particular test script
-#   module_list - input - list of module names
+#   module_list - input - list of module names, for example "numpy;h5py"
```

Comment:
Should state that the quotation marks are necessary, since it's easy to read them as setting apart the example.
```
module_list - input - list of modules names, for example "numpy;h5py" (including the quotation marks)
```

Reply:
Added. Also corrected the "FATAL_ERROR" which was originally intended.